lib/node_modules/@stdlib/blas/base/dnrm2/test/test.dnrm2.js

```diff
@@ -45,14 +76,39 @@ tape( 'the function calculates the L2-norm of a vector', function test( t ) {
 
 	x = new Float64Array( [ 1.0, -2.0, -4.0, 5.0, 0.0, 3.0 ] );
 
-	z = dnrm2( x.length, x, 1 );
+	z = dnrm2( x.length, x, 1, 0 );
 	t.strictEqual( z, sqrt( 55.0 ), 'returns expected value' );
 
 	x = new Float64Array( [ -4.0 ] );
 
-	z = dnrm2( x.length, x, 1 );
+	z = dnrm2( x.length, x, 1, 0 );
 	t.strictEqual( z, 4.0, 'returns expected value' );
 
+	x = new Float64Array([ 1e150, 1e150, 1e150, 1e150 ]);
+
+	z = dnrm2( x.length, x, 1, 0 );
+	t.strictEqual( z, 2.0e+150, 'returns expected value' );
+
+	x = new Float64Array([ 1e-155, 1e-155, 1e-155, 1e-155 ]);
```

Comment:
```suggestion
	x = new Float64Array( [ 1.0e-155, 1.0e-155, 1.0e-155, 1.0e-155 ] );
```

Use decimals for representing double-precision decimal values. Applies here and elsewhere.